index.html

```diff
@@ -111,7 +122,8 @@
               </div>
               <div
                 class="no-drag smartdown-tile"
-                v-bind:id="item.divID">
+                v-bind:id="item.divID"
+                v-bind:class="{ 'in-focus': isSdTileInFocus(item.contentURL.split('/')[1]) }">
```

Comment:
DRY this up!

Instead of repeating `item.contentURL.split('/')[1]` all over the place, I amended the `item` model to include a `.name` property so that we can now do:

```javascript
v-bind:class="{ 'in-focus': isCardInFocus(item.name) }">
```